<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Event::services::qx::RunChangeDialog</class>
 <widget class="QDialog" name="Event::services::qx::RunChangeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Class</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edClassName">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Run ID </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="edRunId">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="specialValueText">
        <string>None</string>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Lock number</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="edLockNumber">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="specialValueText">
        <string>None</string>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="grpFirstName">
     <property name="title">
      <string>First name</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="edFirstNameOrig"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>-&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edFirstName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpLastName">
     <property name="title">
      <string>Last name</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="edLastNameOrig"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>-&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edLastName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpRegistration">
     <property name="title">
      <string>Registration</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLineEdit" name="edRegistrationOrig"/>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>-&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edRegistration"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpSiCard">
     <property name="title">
      <string>SI Card</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QSpinBox" name="edSiCardOrig">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <property name="specialValueText">
         <string>None</string>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="edSiCardRentOrig">
        <property name="text">
         <string>Rent</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>-&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="edSiCard">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <property name="specialValueText">
         <string>None</string>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="edSiCardRent">
        <property name="text">
         <string>Rent</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Note</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edNote">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblError">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Message</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edStatusMessage"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="chkForce">
       <property name="text">
        <string>Force</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btReject">
       <property name="text">
        <string>Reject</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btAccept">
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>grpFirstName</tabstop>
  <tabstop>edFirstNameOrig</tabstop>
  <tabstop>edFirstName</tabstop>
  <tabstop>grpLastName</tabstop>
  <tabstop>edLastNameOrig</tabstop>
  <tabstop>edLastName</tabstop>
  <tabstop>grpRegistration</tabstop>
  <tabstop>edRegistrationOrig</tabstop>
  <tabstop>edRegistration</tabstop>
  <tabstop>grpSiCard</tabstop>
  <tabstop>edSiCardOrig</tabstop>
  <tabstop>edSiCardRentOrig</tabstop>
  <tabstop>edSiCard</tabstop>
  <tabstop>edSiCardRent</tabstop>
  <tabstop>edStatusMessage</tabstop>
  <tabstop>btCancel</tabstop>
  <tabstop>btReject</tabstop>
  <tabstop>btAccept</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btCancel</sender>
   <signal>clicked()</signal>
   <receiver>Event::services::qx::RunChangeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>539</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
